index.js

```diff
@@ -30,7 +33,12 @@ Do the following:
 
    HINT: no function required
 */
-
+let varOne=1;
+let varTwo=2;
+if (varOne<varTwo){
+  varOne=3;
+  console.log(varOne);
+}
 
```

Comment:
Not sure why console.log() is spaced out but glad your console logging! Don't forget to return the value.